Fix "make dist" in `serverboot'.  Not really needed anyway since
`serverboot' is not in the distribution.

diff --git a/serverboot/Makefile b/serverboot/Makefile
index 653d596..233e9b0 100644
--- a/serverboot/Makefile
+++ b/serverboot/Makefile
@@ -25,7 +25,7 @@ SRCS = bootstrap.c ffs_compat.c load.c wiring.c \
 LCLHDRS = assert.h disk_inode_ffs.h fs.h queue.h defs.h \
 	 minix_ffs_compat.h wiring.h dir.h ffs_compat.h minix_fs.h \
 	disk_inode.h file_io.h minix_super.h mach-exec.h
-EXTRA_DIST = def_pager_setup.c default_pager.c kalloc.c
+DIST_FILES = def_pager_setup.c default_pager.c kalloc.c
 target = serverboot
 HURDLIBS = threads
 installationdir = $(bootdir)
